### ISSUE TITLE
Remove Organization Access in Oauth

### ISSUE THREAD

**Issue by BlueBeret** [QasimWani/LeetHub#561]
## Remove Organization Access in Oauth

**Is your feature request related to a problem? Please describe.**
I use my personal github account for work. All codes in my organization are private and are not supposed to be viewed and/or edited by other person and/or third party outside the organization. However, it seems that leethub have access to user organization and I don't think my company will allow that.

**Describe the solution you'd like**
An option to remove organization acess or completely remove organization access in oauth.

**Additional context**
![image](https://github.com/QasimWani/LeetHub/assets/63086194/f8780534-fd6f-4c6f-b55f-491856663a7a)

**Comment by mksonkar** [comment]
Yes, why does it need auth for private repos in the first place? It should only have access the repo where it should push the leetCode problems.
